tests/test_utils_platform_specific.py

```diff
@@ -5,19 +5,25 @@
 
 from tests import BASE_DIR, TempDirTestCase
 
-if not platform.system().lower().startswith('win'):
-    raise unittest.SkipTest('Only available on Windows')
-
-DUMMY_APP_CONTENT = f"""
+if platform.system().lower().startswith('win'):
+    DUMMY_APP_CONTENT = f"""
 import sys
 sys.path.append('{(BASE_DIR.parent / 'src').as_posix()}')
-from notsotuf.utils.windows import start_script_and_exit
-start_script_and_exit(src_dir=sys.argv[1], dst_dir=sys.argv[2])
+from notsotuf.utils.platform_specific import _install_update_win
+_install_update_win(src_dir=sys.argv[1], dst_dir=sys.argv[2])
```

Comment:
Couldn't we get rid of the platform check by importing `install_update`, as follows?

```python
DUMMY_APP_CONTENT = f"""
import sys
sys.path.append('{(BASE_DIR.parent / 'src').as_posix()}')
from notsotuf.utils.platform_specific import install_update
install_update(src_dir=sys.argv[1], dst_dir=sys.argv[2])
"""
```

Reply:
Yes we totally could. Just wrote it like this, because this was how it was originally coded. Let me know if this is what you want.

Reply:
I think it would be neat if this works. It tests the platform logic in `install_update` as well as the platform-specific implementations.